sw/airborne/subsystems/navigation/common_nav.c

```diff
@@ -59,6 +59,41 @@ void compute_dist2_to_home(void)
 #endif
 }
 
+/** Compute time to home
+ * use wind and airspeed when available
+ */
+float get_time_to_home(void)
+{
+  struct FloatVect2 vect_to_home;
+  vect_to_home.x = waypoints[WP_HOME].x - stateGetPositionEnu_f()->x;
+  vect_to_home.y = waypoints[WP_HOME].y - stateGetPositionEnu_f()->y;
+  // get distance to home
+  float dist_to_home = float_vect2_norm(&vect_to_home);
+  if (dist_to_home > 1.f) {
+    // get windspeed or assume no wind
+    struct FloatVect2 wind = { 0.f, 0.f };
+    if (stateIsWindspeedValid()) {
+      wind = *stateGetHorizontalWindspeed_f();
+    }
+    // compute effective windspeed when flying to home point
+    float wind_to_home = (wind.x * vect_to_home.x + wind.y * vect_to_home.y) / dist_to_home;
+    // get airspeed or assume constant nominal airspeed
```

Comment:
The current  set airspeed would be nice eg one has this to be.

   <define name="MINIMUM_AIRSPEED" value="10." unit="m/s"/>
    <define name="NOMINAL_AIRSPEED" value="15." unit="m/s"/> <!-- TODO: Determine -->
    <define name="MAXIMUM_AIRSPEED" value="23." unit="m/s"/> <!-- TODO: Determine -->

    <!-- Values here are only referred to by the flightplan m*_s*_airspeed.xml in final AC -->
    <define name="CLIMB_AIRSPEED" value="12." unit="m/s"/>
    <define name="TRACKING_AIRSPEED" value="18." unit="m/s"/>
    <define name="GLIDE_AIRSPEED" value="10." unit="m/s"/>
    <define name="STALL_AIRSPEED" value="7." unit="m/s"/>
    <define name="RACE_AIRSPEED" value="22." unit="m/s"/>

Reply:
This might be a later addition, but so far only `NOMINAL_AIRSPEED` is available on all airframes